backend/Dockerfile

```diff
@@ -4,12 +4,18 @@ FROM node:12-alpine
 RUN apk update && apk upgrade && apk add --no-cache --virtual builds-deps build-base \
   g++ make python tini
 
-WORKDIR /usr/home/postmangovsg/
-COPY ./package* ./
-RUN npm ci
+WORKDIR /usr/home/postmangovsg/backend
+
+COPY modules /usr/home/postmangovsg/modules
+
+RUN cd /usr/home/postmangovsg/modules/postman-templating && npm install && npm run build
+
+COPY backend/package* ./
+RUN npm install
```

Comment:
Had to npm install here because of this bug: https://npm.community/t/npm-ci-fails-with-transitive-local-packages/4059